.github/data/distros.yml

```diff
@@ -124,10 +124,6 @@ include:
     packages:
       <<: *fedora_packages
       repo_distro: fedora/36
-      arches:
-        - x86_64
-        - armhfp
-        - aarch64
```

Comment:
why do you remove x86_64 and aarch64?

Reply:
computers are dangerous

Reply:
Because this only existed to redefine the array that’s inherited from the Fedora 37 package configuration because of the `<<: *fedora_packages` line just above this.

In general, we’re trying to minimize duplication in this file as much as possible, so we’re making very aggressive use of YAML anchors and map-merge functionality. You can see a similar case below this with Ubuntu (we only build 32-bit x86 packages on Ubuntu 18.04, so it has it’s own redefinition of the `arches` array that the other versions are inheriting from the Ubuntu 22.10 config at the moment).